### ISSUE TITLE
Run and advance command caption doesn't adapt when selecting more than one cell

### ISSUE THREAD

**Issue by HaudinFlorence** [jupyterlab/jupyterlab#14161]
## Description
In a recent PR https://github.com/jupyterlab/jupyterlab/pull/13861, some changes were proposed to adapt the caption associated to the different buttons of the notebook toolbar. Most particularly, when selecting more than one cell, the number of cells is changing in the displayed caption, as shown here (in the case of copy cells command) :
![selection1](https://user-images.githubusercontent.com/99649086/224319585-9d578e47-9020-4f55-b9e3-f5af6a2475e0.png)

Nevertheless, it appears that the caption doesn't adapt for the run and advance command as shown here:
![selection2](https://user-images.githubusercontent.com/99649086/224319984-f63515ac-5780-4eba-bd43-8c6f8ff123ce.png)

## Reproduce

1. Open a notebook
2. Click on the run and advance button (play triangular button) and check the caption: it is "Run this cell and advance"
4. Add n cells and select the n+1 cells of the notebook
5. Click again on the run and advance button
6. See the corresponding caption: it still mentions a single cell

## Expected behavior

When n cells are selected, the caption should be "Run these n cells and advance"

**Comment by MMesch** [comment]
The triaging agent has identified this issue as resolved. I just tested (see below) including running it and everything seems indeed to work fine now. Please reopen/comment in case anything is left to be done.

---

**Agent Rationale**: Code analysis reveals that the dynamic caption functionality is properly implemented in the current codebase at [packages/notebook-extension/src/index.ts:2516-2523](https://github.com/jupyterlab/jupyterlab/blob/09411fab41ac68d9b3619ad487940bedb1597035/packages/notebook-extension/src/index.ts#L2509-L2525). The runAndAdvance command uses the same pattern as the working copy command, with trans._n() and current?.content.selectedCells.length to dynamically update captions. Since the implementation exists and follows the correct pattern, this issue appears to have been resolved in subsequent updates after the original report.

---

<img width="550" height="269" alt="Image" src="https://github.com/user-attachments/assets/046b831d-13ca-4fab-9f40-18f8361a32a5" />
